core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -507,7 +507,8 @@ void core_init_mmu_regs(void)
 	mair |= MAIR_ATTR_SET(ATTR_IWBWA_OWBWA_NTR, ATTR_IWBWA_OWBWA_NTR_INDEX);
 	write_mair_el1(mair);
 
-	tcr  = TCR_XRGNX_WBWA << TCR_IRGN0_SHIFT;
+	tcr = TCR_RES1;
```

Comment:
As I read the TRM for A53 setting it to 0 results in using a reserved value whatever that means. Another option would be to always initialized with `TCR_TG1_4KB`, it's the same bit pattern.